app/src/entities/static/StaticClass.ts

```diff
@@ -0,0 +1,24 @@
+import {Entity, Column} from 'typeorm';
+import {Faction} from '../../constants/faction';
+import {Loadout} from "../../constants/loadout";
+
+@Entity()
+export default class StaticClass {
+
+    // Maps to ingame loadout ID
+    @Column({
+        primary: true,
```

Comment:
Not sure if this is correct. Mongo use `_id` for sharding and therefor it works a little bit different then primary keys in sql dbs